tox.ini

```diff
@@ -11,11 +11,15 @@ deps =
 commands = py.test
 
 [pytest]
-addopts=--cov=uswds_forms --cov-report=term --cov-report=html
+addopts=--cov=uswds_forms --cov-report=term --cov-report=html --ignore=venv
```

Comment:
If users install a virtualenv at `venv` as recommended in the README, then pytest will recurse through that and make tests take *really* long to run (and also probably fail).